### ISSUE TITLE
Code does not display properly on Keras.io (no line breaks)

### ISSUE THREAD

**Issue by jisungk** [fchollet/keras#2332]
The code on a particular page on keras.io does not display properly. There are no line breaks.

[Link to page](http://keras.io/examples/#multilayer-perceptron-mlp-for-multi-class-softmax-classification)

![screenshot from 2016-04-14 15 46 47](https://cloud.githubusercontent.com/assets/9053987/14541473/27fc0eea-0258-11e6-83e0-239cac053d64.png)

**Comment by fchollet** [comment]
Fixed the glitch by removing the pages.

**Comment by jisungk** [comment]
This seems to also persist in other pages:

[http://keras.io/utils/visualization/](http://keras.io/utils/visualization/)
